### PR TITLE
Test Changes in 'Create a focus trap on Help Dialog #19'

### PR DESCRIPTION
**Pull request by nogalpaulina**

### CODE REVIEW

File: components/portal/main/partials/header.html [hunk 1]
```diff
@@ -19,36 +19,47 @@
...
+    md-colors="{ backgroundColor: '{{portal.theme.name}}-primary' }" layout="row" flex-gt-xs="100"
+    layout-align="center center" hide-xs>
+    <search ng-show="APP_FLAGS.showSearch"></search>
+  </div>
+
+  <myuw-help myuw-help-title="Help links" slot="myuw-help" show-button>
+    <div slot="myuw-help-content">
+      <ul>
+        <li>
+          <a href="https://outages.doit.wisc.edu/">Check the Outages page</a>
```
Comment (apetro) [inline components/portal/main/partials/header.html]: The most important thing to say is this is fine demo content and we should just try it.

As we tighten this up towards something to cut a release of and ship, will need to figure out how to make this help content differ in Madison vs System contexts or include links appropriate to both. Cf. the different help links and experiences you get accessing <https://my.wisc.edu> vs <https://my.wisconsin.edu>

Comment (nogalpaulina) [reply components/portal/main/partials/header.html]: I added more meaningful links, found in the footer of MyUW. Would it be helpful to also include the link to https://kb.wisconsin.edu/myuwsystem/page.php?id=15303 or https://kb.wisconsin.edu/myuwsystem/ ? As we have it in the footer of https://my.wisconsin.edu.

**Comment by nogalpaulina** [comment]
> Locally tested by
> 
> ```shell
> npm run static:dev
> ```
> 
> When I click the ❓ icon in the header, the help dialog does not display.
> 
> 2020-01-14-myuw-app-framework-pr-491

This issue has been resolved by adding a fixed position to the help dialog when it's open.

File: components/portal/main/partials/header.html (hunk 1 above)
Comment (apetro) [reply components/portal/main/partials/header.html]: Further updated links in ff9c6b3 .

File: components/head-static.html [hunk 2]
```diff
@@ -35,17 +35,20 @@
...
 <!-- myuw-web-components -->
-<script type="module" src="https://unpkg.com/@myuw-web-components/myuw-app-bar@1.5.4/dist/myuw-app-bar.min.mjs"></script>
+<script type="module"
+  src="https://unpkg.com/@myuw-web-components/myuw-app-bar@1.5.4/dist/myuw-app-bar.min.mjs"></script>
 <script nomodule src="https://unpkg.com/@myuw-web-components/myuw-app-bar@1.5.4/dist/myuw-app-bar.min.js"></script>
 
 <script type="module" src="https://unpkg.com/@myuw-web-components/myuw-banner@1.1.1/dist/myuw-banner.min.mjs"></script>
 <script nomodule scr="https://unpkg.com/@myuw-web-components/myuw-banner@1.1.1/dist/myuw-banner.min.js"></script>
 
+<script type="module" src="http://cdn.my.wisc.edu/@myuw-web-components/myuw-help@1.5.1/myuw-help.min.mjs"></script>
```
Comment (apetro) [inline components/head-static.html]: It turns out this needs to be via `https:`` rather than `http:`; fixed in #942 .

**Pull request merged**